internal/common/environment.go

```diff
@@ -16,76 +16,27 @@ import (
 )
 
 const (
-	OperatorVersionKey        = "OPERATOR_VERSION"
-	TemplateValidatorImageKey = "VALIDATOR_IMAGE"
-	CleanupVMImageKey         = "CLEANUP_VM_IMG"
-	CopyTemplateImageKey      = "COPY_TEMPLATE_IMG"
-	ModifyDataObjectImageKey  = "MODIFY_DATA_OBJECT_IMG"
-	CreateVMImageKey          = "CREATE_VM_IMG"
-	DiskVirtCustomizeImageKey = "DISK_VIRT_CUSTOMIZE_IMG"
-	DiskVirtSysprepImageKey   = "DISK_VIRT_SYSPREP_IMG"
-	ModifyVMTemplateImageKey  = "MODIFY_VM_TEMPLATE_IMG"
-	WaitForVMISTatusImageKey  = "WAIT_FOR_VMI_STATUS_IMG"
-	VirtioImageKey            = "VIRTIO_IMG"
-	GenerateSSHKeysImageKey   = "GENERATE_SSH_KEYS_IMG"
+	OperatorVersionKey          = "OPERATOR_VERSION"
+	TemplateValidatorImageKey   = "VALIDATOR_IMAGE"
+	TektonTasksImageKey         = "TEKTON_TASKS_IMG"
+	TektonTasksDiskVirtImageKey = "TEKTON_TASKS_DISK_VIRT_IMG"
+	VirtioImageKey              = "VIRTIO_IMG"
 
-	DefaultWaitForVMIStatusIMG  = "quay.io/kubevirt/tekton-task-wait-for-vmi-status:" + TektonTasksVersion
-	DeafultModifyVMTemplateIMG  = "quay.io/kubevirt/tekton-task-modify-vm-template:" + TektonTasksVersion
-	DeafultDiskVirtSysprepIMG   = "quay.io/kubevirt/tekton-task-disk-virt-sysprep:" + TektonTasksVersion
-	DeafultDiskVirtCustomizeIMG = "quay.io/kubevirt/tekton-task-disk-virt-customize:" + TektonTasksVersion
-	DeafultCreateVMIMG          = "quay.io/kubevirt/tekton-task-create-vm:" + TektonTasksVersion
-	DeafultModifyDataObjectIMG  = "quay.io/kubevirt/tekton-task-modify-data-object:" + TektonTasksVersion
-	DeafultCopyTemplateIMG      = "quay.io/kubevirt/tekton-task-copy-template:" + TektonTasksVersion
-	DeafultCleanupVMIMG         = "quay.io/kubevirt/tekton-task-execute-in-vm:" + TektonTasksVersion
-	GenerateSSHKeysIMG          = "quay.io/kubevirt/tekton-task-generate-ssh-keys:" + TektonTasksVersion
-	DefaultVirtioIMG            = "quay.io/kubevirt/virtio-container-disk:v0.59.0"
+	DefaultTektonTasksIMG         = "quay.io/kubevirt/tekton-tasks:" + TektonTasksVersion
+	DeafultTektonTasksDiskVirtIMG = "quay.io/kubevirt/tekton-tasks-disk-virt:" + TektonTasksVersion
+	DefaultVirtioIMG              = "quay.io/kubevirt/virtio-container-disk:v0.59.0"
```

Comment:
Btw. is the fixed `v0.59.0` appropriate here?

Reply:
in TTO we have full automation for bumping version of this variable: https://github.com/kubevirt/tekton-tasks-operator/blob/main/.github/workflows/update-virtio-version.yaml